sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.Xunit.Legacy.cs

```diff
@@ -5,7 +5,7 @@
     using Xunit;
     using Xunit.Extensions; // v1.9.1
 
-    class Program
+    public class Program
```

Comment:
Same here, I updated the test code to be closer to valid Xunit test code e.g. making the class public and adding _InlineData_ for the _Theory_ tests (Xunit already has an analyzer that raises an issue if you don't).